src/query/executor.rs

```diff
@@ -38,35 +36,17 @@ pub trait Result: Send + Debug {
     fn into_bytes(self: Box<Self>) -> Vec<u8>;
 }
 
-impl<F: Field> Result for Vec<AdditiveShare<F>>
-where
-    AdditiveShare<F>: Serializable,
-{
-    fn into_bytes(self: Box<Self>) -> Vec<u8> {
-        let mut r = vec![0u8; self.len() * <AdditiveShare<F> as Serializable>::Size::USIZE];
-        for (i, share) in self.into_iter().enumerate() {
-            share.serialize(GenericArray::from_mut_slice(
-                &mut r[i * <AdditiveShare<F> as Serializable>::Size::USIZE
-                    ..(i + 1) * <AdditiveShare<F> as Serializable>::Size::USIZE],
-            ));
-        }
-
-        r
-    }
-}
```

Comment:
I'm surprised you can remove this. Was it unused?

Reply:
Yes and no. 

Mostly no, because I didn't remove it, I generalized the implementation.  Any vector of `Serializable` items can now implement `Result`